app/database/schema.prisma

```diff
@@ -344,6 +344,8 @@ model Location {
   description String?
   address     String?
 
+  imageUrl String?
+
```

Comment:
I think we should add a thumbnail. Rather now than having to fix this later. You can see how we do it with asset images. Once we add a thumbnail we should load that on all lists and filters and so on as its drastically smaller.